featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -121,7 +122,7 @@ def test_init_and_name(es):
     for transform_prim in trans_primitives:
         # skip automated testing if a few special cases
         features_to_use = log_features
-        if transform_prim in [NotEqual, Equal]:
+        if transform_prim in [NotEqual, Equal, BoxCox]:
```

Comment:
curious, why is BoxCox here but not Yeo-Johnson?

Reply:
Put comment here why:
https://github.com/alteryx/featuretools/pull/2387/commits/475efc8709837593cbea0a7194cb90453f25dd6b

Reply:
added comment why